jukejoint_frontend/src/pod/player/components/PlayerDuration.tsx

```diff
@@ -1,16 +1,112 @@
 import React, { Component } from 'react';
+import { AppContextProp, withAppContext, AppContextType } from '../../../toolkit/util/AppContext';
```

Comment:
`PlayerDuration` is already in `PlayerContainer` which is connected to `AppContext`, so you don't have to reconnect it here. Just pass down the `duration` prop from the container, and we can remove all the connection stuff from here.